plugins/woocommerce/client/blocks/assets/js/blocks/product-details/edit.tsx

```diff
@@ -106,8 +109,22 @@ const useIsInvalidQueryLoopContext = ( clientId: string, postType: string ) => {
 const Edit = ( { clientId, context }: ProductDetailsEditProps ) => {
 	const blockProps = useBlockProps();
 
+	const { hasInnerBlocks, wasBlockJustInserted } = useSelect(
+		( select ) => {
+			const blocks = select( blockEditorStore ).getBlocks( clientId );
+			return {
+				hasInnerBlocks: blocks.length > 0,
+				wasBlockJustInserted:
+					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+					// @ts-ignore method exists but not typed
```

Comment:
```suggestion
					// @ts-expected-error method exists but not typed
```

I would use `@ts-expected-error`